sponsorship_compassion/view/contract_view.xml

```diff
@@ -77,7 +77,7 @@
                         </group>
                         <group>
                             <field name="child_id" domain="[('state', 'in', ('N', 'D', 'I', 'R', 'Z')), ('project_id.suspension', 'not in', ('suspended','fund-suspended'))]"
-                                   options="{'create': false, 'm2o_dialog': false, 'create_edit': false}"/>
+                                   options="{'create': false, 'm2o_dialog': false, 'create_edit': false, 'search_more':True, 'field_color':'state', 'colors':{'R':'red', 'F':'gray', 'X':'gray', 'N':'blue'} }"/>

```

Comment:
You can remove F and X colors as they are filtered in the domain and will never appear in selection.